src/lib/test/Project.test.js

```diff
@@ -1,73 +1,50 @@
 import Project from "../Project.js";
+import chatData from "./chatData.json";
```

Comment:
Can you clean up examples that won't have any chat ref?

Reply:
i cleaned up the user messages. can you check again?